Makefile

```diff
@@ -187,6 +226,12 @@ deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in
 undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/config.
 	$(KUSTOMIZE) build config/deploy | kubectl delete -f -
 
+deploy-dependencies: kustomize ## Deploy dependencies to the K8s cluster specified in ~/.kube/config.
+	$(KUSTOMIZE) build config/dependencies | kubectl apply -f -
+	kubectl -n "$(KUADRANT_NAMESPACE)" wait --timeout=300s --for=condition=Available deployments --all
+	kubectl apply -n "$(KUADRANT_NAMESPACE)" -f utils/local-deployment/authorino.yaml
```

Comment:
this is not part of the env setup, this is the job of the kuadrant-controller. 

Do the files `utils/local-deployment/authorino.yaml` and `utils/local-deployment/limitador.yaml` make any sense in this repo?